### DISCUSSION TITLE
posts/must-have-applications/

### DISCUSSION THREAD

**Discussion by giscus** [ZhymabekRoman/zhymabekroman.github.io#1] [General]
# posts/must-have-applications/

My thoughts, projects and etc.

https://zhymabekroman.github.io/posts/must-have-applications/

**Comment by AshutoshGitMirror** [comment]
sorry for hijacking the empty comment section but, 

was freedium repository deleted for good? is the project dead now?

**Reply by ZhymabekRoman** [reply]
Our whole Github organization is not public for now.

Reddit community, that was beginning all of that also gone - [reddit.com/r/paywall/comments/15jsr6z/bypass_mediumcom_paywall](https://www.reddit.com/r/paywall/comments/15jsr6z/bypass_mediumcom_paywall/)

We have moved to Codeberg - [codeberg.org/Freedium-cfd](https://codeberg.org/Freedium-cfd)

**Comment by atengberg** [comment]
[Freedium](https://freedium.cfd) is indeed still available.

Also, please boycott and abstain from Obsidian. A knowledge repository you feed that's closed source? It's counter to the reality that knowledge, like intelligence, is a social phenomena and supporting any organization who does not embrace open source in the pursuit of such a tool is just adding more nails into the coffin of what the internet could have been.

**Reply by AshutoshGitMirror** [reply]
I think logseq fills a similar niche to obsidian, but if there are some extension that are only on obsidian a person might be tempted to use it given obsidian is more popular. 

Also not to get contentious but, I wouldn't truly consider intelligence purely a social phenomena neither would I consider knowledge a purely social phenomena.

**Reply by desaianand1** [reply]
There are many tools like SiYuan, Logseq, Joplin but Obsidian has a great ecosystem and is completely local (offline) if you don't purchase their Sync subscription. I don't see why that's bad. It's not like they can access your data or do anything malicious

**Reply by atengberg** [reply]
Can you prove this though?  "It's not like they can access your data or do anything malicious". 

Moreover, knowledge is foremost open source--people can't help sharing all they've discovered sooner or later. So having a privatized codec for the app of indexing knowledge, runs counter to "essence" of knowledge. This is only true if you accept that knowledge is sharable, in fact--the point of knowledge is sharing it. If you want to get contentious I can argue, consider the case of sharing one's own knowledge with oneself... in fact this is what we all do all the time when we reflect and think on our things, and doing so leads to many a precedent that are then emitted into the greater social informatic tree--eg Plato's Fantasy, Newton's Lawbook, Fanon's Playbook, etc--so it's like choosing to use a medium whose insturmentation is specifically engineered not to be inspectable, when the point of its use is the inspection of reality! 

Perhaps it is just too easy to take for granted. Ahh the slippery slope of wasteful conveince. 

Or let me put it another way, why is it not open source? What's stopping it? It's already a great product, so people will still pay, but with open source then everyone can fully leverage its insturmentation as far apping informartic indexing, and so much more (consider all the marvellous AiPI bindings, etc). 

Also I have a suspicion perhaps one day someone at OHQ will decide to encrypt all your locally saved files after the next update, effectively ransomwaring all your collections arbitrarily. Not a thing you could do, if you didn't egress a complete backup each time. 

Probably my real problem is having my entire "thought process centrally splayed out" seems unideal if I want to preserve my unique creative productive ouput. At some point you'll have enough samples of how someone tends to think, or think they think anyways, you could probably reverse engineer a Turing passable emulation of them, maybe most people don't have to worry about this kind of thing? I think even if you didn't have to worry, the abuses of such technological emulation, ie targetted ad campaigns slowly, incrementally subconciousily your every other spare thought over the course of weeks, months, perhaps even years, to ultimately coerce the route of some decision one way or another you end up making not even having to consider it is perhaps not just your typical act of willful volition, is far more feasible than we want to admit to ourselves is possible.

**Reply by atengberg** [reply]
Ah there must be some 0 day vector into Obsidian, as such untraceable, and then into everyone's heads! Lol.

**Comment by cheeseonamonkey** [comment]
I am curious how you organize your Obsidian vault?

I had some luck with the '[Johnny Decimal System](https://johnnydecimal.com/)' - using it as an entire repository of my life for a few years.

But these days I'm sure there may be some "AI" plugins that can manage a lot of it for you.

**Reply by atengberg** [reply]
Would be cool to connect to immersive VR and then create a literal memory palace of lattic plastered jd ids to help the ability to recreate such visualizing endogenousily unplugged and offline. 

I don't know if this is helpful, but the dichotomy between contiguity and continuity I find to be somewhat imperishable. 

Wow this is really neat, thanks again for sharing!

I'll raise you a JD System Link, with Jimmy Koppel's [How an Ancient Philosophy Problem Explains Software Dependence](https://www.pathsensitive.com/2022/09/bet-you-cant-solve-these-9-dependency.html). Lol everytime I can't remember the name, and regoogling [came across](https://www.sei.cmu.edu/documents/1551/1990_007_001_15821.pdf) [this first](https://essenceofsoftware.com/tutorials/concept-basics/dependency/) this time (but it referenced back to Koppel's original blog). Lol [this](https://repo.darmajaya.ac.id/4764/1/The%20Problem%20With%20Software_%20Why%20Smart%20Engineers%20Write%20Bad%20Code%20%28%20PDFDrive%20%29.pdf) [rabbit hole](https://blog.acolyer.org/2016/09/05/on-the-criteria-to-be-used-in-decomposing-systems-into-modules/) [keeps going](https://tomasp.net/architecture/)! If only there were an _open source_ knowledge cultivating and information refining tool to use!
